src/js/msp/MSPHelper.js

```diff
@@ -726,7 +722,7 @@ MspHelper.prototype.process_data = function(dataHandler) {
                 }
                 break;
             case MSPCodes.MSP_SET_MOTOR:
-                console.log('Motor Speeds Updated');
+                // console.log('Motor Speeds Updated'); // remove repetitive logging
```

Comment:
If we don't need this, please remove the code, better to not left code commented. This log is only shown when modifying the motor speed manually or it is called in a timer manner?

Reply:
Indeed the log is called when using motor testing.

Maybe remove the statistics as I don't think it's useful in user perspective and space is a constraint here as it does not work on small screen.

Reply:
It does not seem very useful, except as curiosity. But at least, we need to hide it when the window is small.

Reply:
Removed visual part. Also why we need configurator build, firmware target and firmware version on the taskbar?  Did not check on Android but it's already beneath the logo besides firmware info is available in cli.